homeassistant/components/tibber/sensor.py

```diff
@@ -414,86 +410,89 @@ def __init__(
         else:
             self._attr_last_reset = None
 
-    async def async_added_to_hass(self):
-        """Start listen for real time data."""
-        self.async_on_remove(
-            async_dispatcher_connect(
-                self.hass,
-                SIGNAL_UPDATE_ENTITY.format(self.unique_id),
-                self._set_state,
-            )
-        )
-
     @property
     def available(self):
         """Return True if entity is available."""
         return self._tibber_home.rt_subscription_running
 
     @callback
-    def _set_state(self, state, timestamp):
-        """Set sensor state."""
-        if (
-            state < self._attr_native_value
-            and self.entity_description.reset_type == ResetType.DAILY
-        ):
-            self._attr_last_reset = dt_util.as_utc(
-                timestamp.replace(hour=0, minute=0, second=0, microsecond=0)
-            )
-        if (
-            state < self._attr_native_value
-            and self.entity_description.reset_type == ResetType.HOURLY
-        ):
-            self._attr_last_reset = dt_util.as_utc(
-                timestamp.replace(minute=0, second=0, microsecond=0)
-            )
+    def _handle_coordinator_update(self) -> None:
+        if not (live_measurement := self.coordinator.get_live_measurement()):  # type: ignore
```

Comment:
Please specify the ignore type to possibly make it easier to resolve in the future.
```py
# type: ignore[bla-bla]
```